### DISCUSSION TITLE
Page level loading

### DISCUSSION THREAD

**Discussion by HartiganHM** [twilio-labs/paste#1531] [Q&A] [answered]
Hello! I was curious as to whether or not Paste has guidance/recommendations for page level loading (e.g. a loading animation, component skeletons, etc.). I know the `Spinner` component exists, but this seems specific for smaller formats. 

Let me know if I can provide any other details. Thank you!

**Comment by zahnster** [comment, marked as answer]
Hi @HartiganHM,

We do have a Skeleton Loader component on our roadmap, scheduled for for our July 13th release.

So, stay tuned! It'll be rolling out soon. :)
